core/src/worker/workflow/workflow_stream.rs

```diff
@@ -165,7 +165,6 @@ impl WFStream {
                         None
                     }
                     WFStreamInput::PollerError(e) => {
-                        warn!("WFT poller errored, shutting down");
```

Comment:
Is there a consequence to removing this warning? Can/should we warn somewhere in general when a fatal worker error occurs, or is the fact that it is returned as an error from poll for lang to warn and start shutdown enough?

Reply:
It's actually just redundant. I removed it because there's another line that will always log for the same reason.